integration_tests/mockApis/prisonerListApi.ts

```diff
@@ -17,283 +18,9 @@ const stubPrisonerList = (prisonId = 'BXI', page = 0, pageSize = 9999): SuperAge
-            },
-          },
-          {
-            prisonerNumber: 'A5502DZ',
-            bookingId: '1209043',
-            bookNumber: '45919A',
-            firstName: 'JOHN',
-            lastName: 'NOAKES',
-            dateOfBirth: '1967-01-02',
-            gender: 'Male',
-            youthOffender: false,
-            status: 'ACTIVE IN',
-            lastMovementTypeCode: 'ADM',
-            lastMovementReasonCode: 'I',
-            inOutStatus: 'IN',
-            prisonId: 'BXI',
-            prisonName: 'Brixton (HMP)',
-            cellLocation: 'A-1-027',
-            aliases: [],
-            alerts: [],
-            legalStatus: 'SENTENCED',
-            imprisonmentStatus: 'ADIMP_ORA20',
-            imprisonmentStatusDescription: 'ORA 2020 Standard Determinate Sentence',
-            mostSeriousOffence: 'Abandon a fighting dog',
-            recall: false,
-            indeterminateSentence: false,
-            sentenceStartDate: '2023-07-26',
-            releaseDate: '2029-07-18',
-            sentenceExpiryDate: '2035-07-18',
-            licenceExpiryDate: '2035-07-18',
-            nonDtoReleaseDate: '2029-07-18',
-            nonDtoReleaseDateType: 'CRD',
-            receptionDate: '2023-07-26',
-            conditionalReleaseDate: '2029-07-18',
-            locationDescription: 'Brixton (HMP)',
-            restrictedPatient: false,
-            currentIncentive: {
-              level: {
-                code: 'STD',
-                description: 'Standard',
-              },
-              dateTime: '2023-07-26T11:13:38',
-              nextReviewDate: '2023-10-26',
-            },
-          },
-          {
-            prisonerNumber: 'A8335DY',
-            bookingId: '1201879',
-            bookNumber: '38985A',
-            firstName: 'NORMAAX',
-            middleNames: 'STANLEAX',
-            lastName: 'FLETCHEAX',
-            dateOfBirth: '2000-01-01',
-            gender: 'Male',
-            youthOffender: false,
-            status: 'ACTIVE IN',
-            lastMovementTypeCode: 'ADM',
-            lastMovementReasonCode: 'INT',
-            inOutStatus: 'IN',
-            prisonId: 'BXI',
-            prisonName: 'Brixton (HMP)',
-            cellLocation: 'RECP',
-            aliases: [],
-            alerts: [],
-            legalStatus: 'SENTENCED',
-            imprisonmentStatus: 'SENT03',
-            imprisonmentStatusDescription: 'Adult Imprisonment Without Option CJA03',
-            recall: false,
-            indeterminateSentence: false,
-            receptionDate: '2022-02-17',
-            locationDescription: 'Brixton (HMP)',
-            restrictedPatient: false,
-            currentIncentive: {
-              level: {
-                code: 'STD',
-                description: 'Standard',
-              },
-              dateTime: '2022-02-18T13:00:59',
-              nextReviewDate: '2023-02-18',
-            },
-          },
-        ],
+        content: Object.values(prisoners)
```

Comment:
Related to this change (but not 100% necessary), I've changed the stub data for the prisoner list to be prisoners from the main stub prisoner list data that match the prison ID used in the request; rather than a hardcoded list o prisoners as before